### DISCUSSION TITLE
Shouldn't the length of the string be randomly chosen between the minimum and maximum number of characters?

### DISCUSSION THREAD

**Discussion by kycasdzxc** [yayakm/RegexStringGenerator#22] [Q&A] [answered]
Currently, with the method generator.generateString(10, 20), it receives the minimum and maximum number of characters. However, it seems that the generated string always ends up being of the maximum length.

Shouldn't the length of the string be randomly chosen between the minimum and maximum number of characters? If you agree with this, I will register it as an issue.

![image](https://github.com/user-attachments/assets/ec4a7c2d-5156-4261-bd8c-59208d03d516)

**Comment by yayakm** [comment, marked as answer]
I've just checked and indeed the length of the generated string is always equal to the maximum length, you can register this as an issue.

**Reply by kycasdzxc** [reply]
> I've just checked and indeed the length of the generated string is always equal to the maximum length, you can register this as an issue.

ok, thank you.
